executor/audioclip_text.py

```diff
@@ -6,6 +6,7 @@
 
 import torch
 from jina import DocumentArray, Executor, requests
+from jina.logging.predefined import default_logger
```

Comment:
please do not import a default logger, Jina Executors already come with a loggee that u can use with self.logger